### PR TITLE
chore: updated changelog file for v0.5.1

### PR DESCRIPTION
**Pull request by PacoDw**

### CODE REVIEW

**Review by marinsalinas — APPROVED** [review]
LGTM! Just we need @themantissa's  check!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
-## 0.6.0 (Unreleased)
-## 0.5.0 (April 22, 2020)
+## 0.5.1 (Unreleased)
+
+**Implemented enhancements:**
+
+- Support new private and privateSrv connection strings [\#183](https://github.com/terraform-providers/terraform-provider-mongodbatlas/issues/183)
```
Comment (themantissa) [inline CHANGELOG.md]: remove these lines - they don't apply to the hotfix.  Then we are ready!

Comment (themantissa) [reply CHANGELOG.md]: @PacoDw if you can cut these and submit the commit I can approve.

**Review by themantissa — APPROVED** [review]
LGTM!  Nice!

**Pull request merged**